scripts/jenkins/track-upstream.sh

```diff
@@ -31,10 +31,7 @@ case $OBS_TYPE in
             Cloud:OpenStack:Rocky*)
                 OSC_BUILD_DIST=SLE_12_SP4
                 ;;
-            Cloud:OpenStack:Stein*)
-                OSC_BUILD_DIST=SLE_15
-                ;;
-            Cloud:OpenStack:Master)
+            Cloud:OpenStack:Stein*|Cloud:OpenStack:Train*)
                 OSC_BUILD_DIST=SLE_15
```

Comment:
At some point I'm guessing we will split this to allow Train on SLE_15_SP1, but I think that will happen later on.